operations/accounts.md

```diff
@@ -62,3 +62,403 @@ Merges two or more accounts of the same account type together. The given source
+| `AdditionalTaxIdentifier` | [String update value](_objects.md#string-update-value) | optional | Additional tax identifier \(or `null` if the additional tax identifier should not be updated\). |
+| `Contact` | [String update value](_objects.md#string-update-value) | optional | Contact \(or `null` if the contact should not be updated\). |
+| `ContactPerson` | [String update value](_objects.md#string-update-value) | optional | Contact person \(or `null` if the contact person should not be updated\). |
+| `FiscalIdentifier` | [String update value](_objects.md#string-update-value) | optional | Fiscal identifier \(or `null` if the firscal identifier should not be updated\). |
+| `Iata` | [String update value](_objects.md#string-update-value) | optional | Iata \(or `null` if the iata should not be updated\). |
+| `Telephone` | [String update value](_objects.md#string-update-value) | optional | Telephone \(or `null` if the telephone should not be updated\). |
+| `SourceId` | [String update value](_objects.md#string-update-value) | optional | [Source](sources.md#source) \(or `null` if the source should not be updated\). |
+| `Options` | [Company update options](#company-update-options) | optional | Options of the company. |
+| `Classifications` | [Company update classifications](#company-update-classifications) | optional | Options of the company. |
+
+#### Company update options
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Invoiceable` | [Bool update value](_objects.md#bool-update-value) | optional | Invoiceable \(or `null` if the value should not be updated\). |
+| `AddFeesToInvoices` | [Bool update value](_objects.md#bool-update-value) | optional | Add fees to invoices \(or `null` if the value should not be updated\). |
+
+#### Company update classifications
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Corporate` | [Bool update value](_objects.md#bool-update-value) | optional | Corporate \(or `null` if the value should not be updated\). |
+| `Internal` | [Bool update value](_objects.md#bool-update-value) | optional | Internal \(or `null` if the value should not be updated\). |
+| `Private` | [Bool update value](_objects.md#bool-update-value) | optional | Private \(or `null` if the value should not be updated\). |
+| `OnlineTravelAgency` | [Bool update value](_objects.md#bool-update-value) | optional | Online travel agency \(or `null` if the value should not be updated\). |
+| `GlobalDistributionSystem` | [Bool update value](_objects.md#bool-update-value) | optional | Global distribution system \(or `null` if the value should not be updated\). |
+| `Marketing` | [Bool update value](_objects.md#bool-update-value) | optional | Marketing \(or `null` if the value should not be updated\). |
+| `Inactive` | [Bool update value](_objects.md#bool-update-value) | optional | Inactive \(or `null` if the value should not be updated\). |
+
+### Response
+
+```javascript
```

Comment:
@VBuben Are the data for Customer and Company deliberately left blank in the response JSON, or are you intending to add them?